pkg/plugin/admissioncontrollers/webhook/webhook.go

```diff
@@ -80,6 +92,44 @@ func main() {
 		noLabelNamespaces = ""
 	}
 
+	bypassAuth := defaultBypassAuth
+	bypassAuthEnv, ok := os.LookupEnv(admissionControllerBypassAuth)
+	if ok {
+		parsed, err := strconv.ParseBool(bypassAuthEnv)
+		if err != nil {
+			log.Logger().Warn("Unable to parse value, using default",
+				zap.String("env var", admissionControllerBypassAuth),
+				zap.Bool("default", defaultBypassAuth))
+		} else {
+			bypassAuth = parsed
+		}
+	}
+	systemUsers, ok := os.LookupEnv(admissionControllerSystemUsers)
+	if !ok {
+		systemUsers = defaultSystemUsers
+	}
+	externalUsers, ok := os.LookupEnv(admissionControllerExternalUsers)
+	if !ok {
+		externalUsers = ""
+	}
+	externalGroups, ok := os.LookupEnv(admissionControllerExternalGroups)
+	if !ok {
+		externalUsers = ""
```

Comment:
This should be `externalGroups` if I am correct.
Can we add some unit tests so we can catch things like this?

Reply:
Code was extracted to a function & unit tests were added